File: snippets/proxy/add-phone-number/add-phone-number.py
# Download the helper library from https://www.twilio.com/docs/python/install
from twilio.rest import Client

# Your Account SID and Auth Token from twilio.com/console
account_sid = 'ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
auth_token = 'xxxxxxxxxxxxxxxxxxxxxxxxxxx'
client = Client(account_sid, auth_token)

proxy_sid = 'KSxxxxxxxxxxxxxxxxxxxxxxxxxx'

phone_number = client.proxy.services(proxy_sid) \
                          .phone_numbers \
                          .create(sid='PNxxxxxxxxxxxxxxxxxxxxxxx')

print(phone_number.sid)

File: snippets/proxy/add-phone-number/add_phone_number.java
package proxy;

import com.twilio.Twilio;
import com.twilio.rest.proxy.v1.service.PhoneNumber;

public class add_phone_number {

    private static final String ACCOUNT_SID = System.getenv("ACCOUNT_SID");
    private static final String AUTH_TOKEN = System.getenv("AUTH_TOKEN");

    public static void main(String[] args) {
        Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

        PhoneNumber phoneNumber = 
            PhoneNumber.creator(System.getenv("PROXY_SERVICE_SID"))
            .setSid("PNxxxxxxxxxxxxxxxxxxxxxxx")
            .create();
        System.out.println("+ Into the Proxy service, added phone number: " + phoneNumber.getSid());
    }
}
